### PR TITLE
#2319 support default definitions in CSV sink connector

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2319 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-kafka/src/main/java/com/island/ohara/kafka/connector/csv/CsvSinkConnector.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import java.util.stream.Stream;
+
+public abstract class CsvSinkConnector extends RowSinkConnector {
+
+  /**
+   * Define the configuration for the connector.
+   *
+   * @return The SettingDef for this connector.
+   */
+  protected List<SettingDef> _definitions2() {
```
Comment (chia7712) [inline ohara-kafka/src/main/java/com/island/ohara/kafka/connector/csv/CsvSinkConnector.java]: Could we have another name for this method?

Comment (eechih) [reply ohara-kafka/src/main/java/com/island/ohara/kafka/connector/csv/CsvSinkConnector.java]: I want to maintain the same format (starting with underline, the same rules as RowSinkConnector), and have same meaning as the original. So it is quite difficult. :(
Do you have any suggestion? @chia7712

Comment (chia7712) [reply ohara-kafka/src/main/java/com/island/ohara/kafka/connector/csv/CsvSinkConnector.java]: It is ok to remove the "final" from Row connector definition(), and then this new connector is able to override it to have _definition() be abstract.

Comment (eechih) [reply ohara-kafka/src/main/java/com/island/ohara/kafka/connector/csv/CsvSinkConnector.java]: Address your comment. Please review again, thanks. @chia7712

**Pull request merged**